README.md

```diff
@@ -0,0 +1,33 @@
+[](https://img.shields.io/badge/Microverse-blueviolet)
+
+#   Desktop version Portfolio
+
+> The project is aimed at making a desktop responsive potfolio
+
+
+## Built With
+
+-Major languages
+HTML,CSS
+
```

Comment:
[OPTIONAL] -   - Well done adding a README to make a professional readme file; kindly use the [Microverse README template ](https://github.com/microverseinc/readme-template/blob/master/README.md) and feel free to customize the sections, and please fill many sections as you can.